test/quantum_test.exs

```diff
@@ -79,7 +90,7 @@ defmodule QuantumTest do
     for name <- job_names do
       spec = "* * * * *"
       fun = fn -> :ok end
-      job = %Quantum.Job{name: name, schedule: spec, task: fun}
+      job = %Quantum.Job{name: name, schedule: spec, task: fun, nodes: Quantum.Normalizer.default_nodes}
```

Comment:
Now we have to explicitly pass the nodes default value for tests that validate job as a struct.